tools/device-listing/device-listing.cs

```diff
@@ -47,6 +47,8 @@ class Program
             "color",
             "led",
             "spi",
+            "nfc",
```

Comment:
fine to have both categories but since they currently have exactly the same bindings just display one of them